### DISCUSSION TITLE
unclip_ratio工作原理

### DISCUSSION THREAD

**Discussion by djdll** [PaddlePaddle/PaddleOCR#13417] [Q&A]
有没有大佬懂unclip_ratio工作原理，这个参数让我觉的很疑惑，输出的结果不应该是mask计算得到的嘛，为什么会有这么一个参数，而且对检测影响非常之大

**Comment by SWHL** [comment]
去看DBNet论文 → [arxiv](https://arxiv.org/abs/1911.08947)
